Shops/ShopGoblinTinkerer.cs

```diff
@@ -134,234 +118,216 @@ public void InitShop(string currentShop)
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
                     }
 
                     if (NPC.downedMechBoss2)
                     {
                         shop.item[nextSlot].SetDefaults(ItemID.PaladinsShield);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
+                        
                         shop.item[nextSlot].SetDefaults(ItemID.PutridScent);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
+                        
                         shop.item[nextSlot].SetDefaults(ItemID.SharkToothNecklace);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
+                        
                         shop.item[nextSlot].SetDefaults(ItemID.StarCloak);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
+                        
                         shop.item[nextSlot].SetDefaults(ItemID.SunStone);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
+                        
                         shop.item[nextSlot].SetDefaults(ItemID.TitanGlove);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
+                        
                     }
 
 
                     if (NPC.downedMechBoss3)
                     {
                         shop.item[nextSlot].SetDefaults(ItemID.ApprenticeScarf);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
+                        
                         shop.item[nextSlot].SetDefaults(ItemID.SquireShield);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
+                        
                         shop.item[nextSlot].SetDefaults(ItemID.HuntressBuckler);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
+                        
                         shop.item[nextSlot].SetDefaults(ItemID.MonkBelt);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
+                        
                         shop.item[nextSlot].SetDefaults(ItemID.HerculesBeetle);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
+                        
                         shop.item[nextSlot].SetDefaults(ItemID.NecromanticScroll);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
+                        
                         shop.item[nextSlot].SetDefaults(ItemID.PygmyNecklace);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
```

Comment:
You need `++` otherwise everything will be merged into one slot.

Please review the prices.